src/components/Anchor/stories.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+
+import Anchor from ".";
+
+storiesOf("Anchor", module).add("Anchor default", () => (
+  <Anchor hyperLink="https://example.com">Anchor example</Anchor>
```

Comment:
it may be a case of just having an optional prop `component={Link}` but have a look if there's any docs/examples that would help